translations/README.md

```diff
@@ -14,44 +14,42 @@ Refer to GitHub docs for more information on [proposing changes via pull request
 
 A good way to assure the quality of the translations is to build the docs with the up-to-date message catalog (.po file). If you are willing to down that way, follow the steps below.
 
-**Note:** All instructions below assume you are inside `translations/` directory.
+**Note:** All instructions below assume you are running from the project's root directory.
 
-1. Install dependencies (hint: add `--user` flag for a per-user installation, without requiring admin privileges):
+1. Install `tox` (hint: add `--user` flag for a per-user installation, without requiring admin privileges):
 ```
-pip install -r ../requirements.txt
+pip install tox
 ```
 
-2. Copy your PO file to `po/` (replace `<lang>` accordingly):
+2. Copy your .po file to `translations/po/` (replace `<lang>` accordingly):
 ```
-cp path/to/your/translated/file.po po/<lang>.po
+cp path/to/your/translated/file.po translations/po/<lang>.po
 ```
 
-3. If you are adding new translation for a language, add your language code to `LINGUAS` in alphabetical order; otherwise, skip this step.
+3. If you are adding new translation for a language, add your language code to the `LINGUAS` file in alphabetical order; otherwise, skip this step.
```

Comment:
Should be `translations/LINGUAS`. And referencing https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-language would be useful.

Reply:
Done. I mentioned "See supported language codes" and linked to "languages" config's link.